pip/tests/test_re.py

```diff
@@ -100,19 +100,31 @@ def test_qsharp_estimation_with_multiple_params() -> None:
     params = EstimatorParams(3)
     params.items[0].qubit_params.name = QubitParams.GATE_US_E3
     params.items[0].error_budget = 0.333
+    params.items[0].estimate_type = "singlePoint"
```

Comment:
Hmm... I think the fact that this test had to change is a strong indication that we don't want to change the default in Python. This pattern here is used in notebook samples and docs pages, which means this would be a **breaking change** for Python consumers.

Reply:
Sure. This is a breaking change for code we just released. It is not great to make the change the next day. But if we ever want to switch it could be the only option. So, let us wait for more news today to make the final decision: if we want the frontier by default anywhere either in Python or in VS Code.

Reply:
agreed on defaults. reverted to agreed ones